test/functional/test_site/requirements/createModal.md

```diff
@@ -0,0 +1,3 @@
+<modal large header="Establishing Requirements" id="modal:bugRepro">
```

Comment:
Could leave a comment saying this is used by `test_site/testIncludeMultipleModals.md`
```
<!-- Used in ....---->
```